main.go

```diff
@@ -79,14 +79,12 @@ func handlerHelp(ctx context.Context, telegramBot *bot.Bot, update *models.Updat
 }
 
 func handlerSendPhoto(ctx context.Context, tgBot *bot.Bot, update *models.Update) {
-	// TODO: Set actual logic whenever the message is a reply lmao
-	/*
-		    if update.Message.ReplyToMessage != nil {
-			handlerSendPhoto(ctx, bot, update)
-		    } else {
-
-		    }
-	*/
+	
+	if isReply(update) {
+		// if is a reply, send a totally random cat as a quote to the replied message
+		log.Print("-- Detected Reply: fetching random cat from:" + "https://cataas.com/cat")
```

Comment:
There's no need to have logs in here, we're trying to keep the logs output as low as possible.